bigframes/ml/compose.py

```diff
@@ -107,13 +111,13 @@ def _extract_from_bq_model(
         """Extract transformers as ColumnTransformer obj from a BQ Model. Keep the _bqml_model field as None."""
         assert "transformColumns" in bq_model._properties
 
-        transformers: List[
+        transformers_set: Set[
             Tuple[
                 str,
                 Union[preprocessing.PreprocessingType, impute.SimpleImputer],
                 Union[str, List[str]],
             ]
-        ] = []
+        ] = set()
```

Comment:
what's the reason for going from list to set?

Reply:
The processor is different from others. Where previous are 1 -> 1 transform, poly_features is n -> m. So other transformers we get 1 transformed_column, but poly_features we get m transformed_column. Which need to be deduped. And the order of the transformers doesn't matter.